src/components/Edit/NatureOfBusiness.vue

```diff
@@ -1,11 +1,13 @@
 <template>
-  <div id="nature-of-business-info">
+  <div id="nature-of-business-shared">
```

Comment:
This is actually not the shared component. I've been trying to have the ID the same as the filename.

I realize "nature-of-business" ID is already taken by the shared component. I don't have a suggestion on what to call this. I avoided this altogether by not wrapping this in a div, hence no ID 😁 

https://github.com/bcgov/business-create-ui/blob/main/src/components/Registration/NatureOfBusiness.vue

Reply:
You are right! 

Just removed the wrapper, it wasn't required.
We use the id on the view level for our scroll-to validation.

Reply:
I'm not sure what happens to the class binding on the component itself but it wouldn't recognize it without a root node.
I can look into it some more, but my conditional selector wouldn't work without.

Reply:
Will 👀

Reply:
Ok, like the wrapper below. The id's were in conflict with one overriding the other. 
Removed the override ID's (which were just used in validations) and relied on the component template Id's. 
Removed the wrappers, looks good!